dwave/system/composites/reversecomposite.py

```diff
@@ -145,16 +149,25 @@ def sample(self, bqm, anneal_schedules=None, **parameters):
             if "answer_mode" in child.parameters:
                 parameters['answer_mode'] = 'histogram'
 
-        vectors = {}
+        samplesets = None
         for schedule_idx, anneal_schedule in enumerate(anneal_schedules):
             sampleset = child.sample(bqm, anneal_schedule=anneal_schedule, initial_state=initial_state,
                                      **parameters)
 
-            # update vectors
             initial_state, _ = dimod.as_samples(initial_state)
-            vectors = _update_data_vector(vectors, sampleset,
-                                          {'initial_state': [initial_state[0]] * len(sampleset.record.energy),
-                                           'schedule_index': [schedule_idx] * len(sampleset.record.energy)})
+
+            if 'initial_state' not in sampleset.record.dtype.names:
+                vect = [initial_state[0]] * len(sampleset.record.energy)
```

Comment:
Please note this creates a list of shallow copies, a list of items all referencing the same numpy array! Is that safe to do in this context?

Reply:
I don't think that users should ever be changing the initial_state... but you're right, it would be safer to do a deep copy just in case. Will make this change. Thanks!